src/math/lp/lp_core_solver_base.h

```diff
@@ -18,592 +18,595 @@ Revision History:
-    }
-
-    
-    template <typename K>
-    static void swap(vector<K> &v, unsigned i, unsigned j) {
-        auto t = v[i];
-        v[i] = v[j];
-        v[j] = t;
-    }
-        
-    // called when transposing row i and ii
-    void transpose_basis(unsigned i, unsigned ii) {
-        swap(m_basis, i, ii);
-        swap(m_basis_heading, m_basis[i], m_basis[ii]);
-    }
-
-    bool column_is_in_inf_set(unsigned j) const {
-        return m_inf_set.contains(j);
-    }
-
-    bool column_is_base(unsigned j) const {
-        return m_basis_heading[j] >= 0;
-    }
-
-    
-    void update_x_with_feasibility_tracking(unsigned j, const X & v) {
-        TRACE("lar_solver", tout << "j = " << j << ", v = " << v << "\n";);
-        m_x[j] = v;
-        track_column_feasibility(j);
-    }
-
-    void add_delta_to_x_and_track_feasibility(unsigned j, const X & del) {
-        TRACE("lar_solver", tout << "del = " << del << ", was x[" << j << "] = " << m_x[j] << "\n";);
-        m_x[j] += del;
-        TRACE("lar_solver", tout << "became x[" << j << "] = " << m_x[j] << "\n";);
-        track_column_feasibility(j);
-    }
-
-    void update_x(unsigned j, const X & v) {
-        TRACE("lar_solver", tout << "j = " << j << ", v = " << v << "\n";);
-        m_x[j] = v;
-    }
-
-    void add_delta_to_x(unsigned j, const X & delta) {
-        TRACE("lar_solver", tout << "j = " << j << ", delta = " << delta << "\n";);
-        m_x[j] += delta;
-    }
-   
-    void track_column_feasibility(unsigned j) {
-        if (column_is_feasible(j))
-            remove_column_from_inf_set(j);
-        else
-            insert_column_into_inf_set(j);
-    }
-    void insert_column_into_inf_set(unsigned j) {
-        TRACE("lar_solver", tout << "j = " << j << "\n";);
-        m_inf_set.insert(j);
-        lp_assert(!column_is_feasible(j));
-    }
-    void remove_column_from_inf_set(unsigned j) {
-        TRACE("lar_solver", tout << "j = " << j << "\n";);
-        m_inf_set.erase(j);
-        lp_assert(column_is_feasible(j));
-    }
-
-    void resize_inf_set(unsigned size) {
-        TRACE("lar_solver",);
-        m_inf_set.resize(size);
-    }
-
-    void clear_inf_set() {
-        TRACE("lar_solver",);
-        m_inf_set.clear();
-    }
-    
-    bool costs_on_nbasis_are_zeros() const {
-        lp_assert(this->basis_heading_is_correct());
-        for (unsigned j = 0; j < this->m_n(); j++) {
-            if (this->m_basis_heading[j] < 0)
-                lp_assert(is_zero(this->m_costs[j]));
-        }
-        return true;
-    }
-    unsigned & iters_with_no_cost_growing() {
-        return m_iters_with_no_cost_growing;
-    }
-
-    const unsigned & iters_with_no_cost_growing() const {
-        return m_iters_with_no_cost_growing;
-    }
-
-    
-    unsigned get_base_column_in_row(unsigned row_index) const {
-        return m_basis[row_index];
-    }
+    return true;
+  }
+
+  bool column_has_lower_bound(unsigned j) const {
```

Comment:
this is not a real change. It is a formatting change. it should not be part of pull request.